src/com/telesign/util/TeleSignRequest.java

```diff
@@ -29,353 +29,381 @@
-	 */
-	public TeleSignRequest(String base, String resource, String method, String customer_id, String secret_key) {
-
-		this.base = base;
-		this.resource = resource;
-		this.customer_id = customer_id;
-		this.secret_key = secret_key;
-
-		post = (method.toLowerCase().equals("post"));
-
-		ts_headers = new TreeMap<String, String>();
-		headers = new TreeMap<String, String>();
-		params = new HashMap<String, String>();
-	}
-
-	/**
-	 * Adds an HTTP 1.1 request header field/value pair to the set of request
-	 * headers.
-	 *
-	 * @param key
-	 *			[Required] A string representing a Request header field
-	 *			<em>name</em>.
-	 * @param value
-	 *			[Required] A string representing a Request header field
-	 *			<em>value</em>.
-	 */
-	public void addHeader(String key, String value) {
-
-		// Check to see if this request header field is a "TeleSign-specific" header field.
-		if ((key.length() > 4) && (key.toLowerCase().substring(0, 5).equals("x-ts-"))) {
-
-			// If using the TeleSign-specific date header, then use a blank line for the standard Date Request header.
-			if (key.toLowerCase().equals("x-ts-date")) {
-
-				ts_date = true;
-			}
-
-			ts_headers.put(key, value);
-		}
-
-		headers.put(key, value);
-	}
-
-	/**
-	 * Adds an HTTP 1.1 Request parameter attribute/value pair to the set of
-	 * Request Parameters.
-	 *
-	 * @param key
-	 *			[Required] A string representing a Request parameter attribute
-	 *			<em>name</em>.
-	 * @param value
-	 *			[Required] A string representing a Request parameter attribute
-	 *			<em>value</em>.
-	 */
-	public void addParam(String key, String value) {
-
-		params.put(key, value);
-	}
-
-	/**
-	 * Sets the entity body of an HTTP 1.1 POST Request message. Contains the
-	 * Request parameters for <strong>Verify.call</strong> and
-	 * <strong>Verify.sms</strong>.
-	 *
-	 * @param post_body
-	 *			[Required] A string that contains the contents of the POST
-	 *			Request's entity body.
-	 */
-	public void setPostBody(String post_body) {
-
-		body = post_body;
-	}
-
-	/**
-	 * Gets the entity body of an HTTP 1.1 POST Request message. Used for calls
-	 * to <strong>Verify.call</strong> and <strong>Verify.sms</strong>.
-	 *
-	 * @return A string that contains the contents of the POST Request's entity
-	 *		 body.
-	 */
-	public String getPostBody() {
-
-		return body;
-	}
-
-	/**
-	 * Returns the set of <em>TeleSign-specific</em> Request header fields.
-	 *
-	 * @return A sorted key/value mapping that contains the TeleSign-specific
-	 *		 Request header fields.
-	 */
-	public Map<String, String> getTsHeaders() {
-
-		return ts_headers;
-	}
-
-	/**
-	 * Returns the set of both the standard <u>and</u> the
-	 * <em>TeleSign-specific</em> Request header fields.
-	 *
-	 * @return A sorted key/value mapping that contains all of the Request header fields.
-	 */
-	public Map<String, String> getAllHeaders() {
+    private static Logger logger = Logger.getLogger(TeleSignRequest.class);
```

Comment:
Github diffs are useless if you touch every single whitespace in the file - hard to see what changed :)